Header header
float64 position0
float64 position1
float64 speed
float64 voltage
float64 current
float64 external_voltage
uint8 digital_inputs
